### ISSUE TITLE
Issue with Google Music Manager Cask

### ISSUE THREAD

**Issue by winkelsdorf** [caskroom/homebrew-cask#21657]
[Google Music Manager](https://github.com/caskroom/homebrew-cask/blob/master/Casks/music-manager.rb) currently fails to install because it's unable to handle correct permissions requiring sudo for installing it's preference pane.

This should probably be escalated together with a similar problem with Parallels from #18638, as it looks like something broke the sudo handling, probably related to the new installation behavior from #13201 and #13966.

This single issue has low priority for me, but after this is the 2nd cask running into sudo issues with the new installation behavior, it's probably worth having a look into..

**Bug description for this particular Cask**

I removed all leftovers of previous installation and received this upon Installation via Cask:

```console
 ~/Dropbox/Application Support/dotfiles/Update  brew cask install music-manager  
==> Downloading https://dl.google.com/dl/androidjumper/mac/2431116/musicmanager.dmg
Already downloaded: /Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg
==> Verifying checksum for Cask music-manager
==> Moving App 'MusicManager.app' to '/Applications/Music Manager.app'
Error: It seems the Preference Pane source is not there: '/opt/homebrew-cask/Caskroom/music-manager/1.0.243.1116/MusicManager.app/Contents/Helpers/MusicManager.prefPane'
```

Like with Parallels, sudoing solves this:
```console
 ✘  ~/Dropbox/Application Support/dotfiles/Update  sudo brew cask install music-manager   
Password:
==> Downloading https://dl.google.com/dl/androidjumper/mac/2431116/musicmanager.dmg
Already downloaded: /Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg
==> Verifying checksum for Cask music-manager
==> It seems there is already an App at '/Applications/Music Manager.app'; not moving.
==> Moving Preference Pane 'MusicManager.prefPane' to '/Users/winkelsdorf/Library/PreferencePanes/MusicManager.prefPane'
🍺  music-manager staged at '/opt/homebrew-cask/Caskroom/music-manager/1.0.243.1116' (114 files, 1,6M)
```

**Scenario 2: Uninstallation without sudo crashes, when previously installed *with* sudo**

```console
 ~/Dropbox/Application Support/dotfiles/Update  brew cask uninstall music-manager   
==> Removing App: '/Applications/Music Manager.app'
==> Removing Preference Pane: '/Users/winkelsdorf/Library/PreferencePanes/MusicManager.prefPane'
Error: Permission denied - /Users/winkelsdorf/Library/PreferencePanes/MusicManager.prefPane/Contents/_CodeSignature/CodeResources
  Most likely, this means you have an outdated version of homebrew-cask. Please run:

      brew uninstall --force brew-cask; brew untap phinze/cask; brew update; brew cleanup; brew cask cleanup

  If this doesn’t fix the problem, please report this bug:

      https://github.com/caskroom/homebrew-cask#reporting-bugs

/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1439:in `unlink'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1439:in `block in remove_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1444:in `platform_support'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1438:in `remove_file'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1427:in `remove'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:770:in `block in remove_entry'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1477:in `block (2 levels) in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1477:in `block (2 levels) in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1477:in `block (2 levels) in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1481:in `postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1476:in `block in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1476:in `block in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1476:in `block in postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:1475:in `postorder_traverse'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:768:in `remove_entry'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:626:in `block in rm_r'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:622:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/fileutils.rb:622:in `rm_r'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:570:in `rmtree'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:101:in `delete'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:34:in `block in uninstall_phase'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:43:in `block in each_artifact'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:42:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:42:in `each_artifact'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:31:in `uninstall_phase'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:298:in `block in uninstall_artifacts'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:295:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:295:in `uninstall_artifacts'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:286:in `uninstall'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/uninstall.rb:12:in `block in run'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/uninstall.rb:6:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/uninstall.rb:6:in `run'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:83:in `run_command'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:121:in `process'
/usr/local/Library/Taps/caskroom/homebrew-cask/cmd/brew-cask.rb:26:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Library/brew.rb:22:in `require?'
/usr/local/Library/brew.rb:93:in `<main>'
```

**Full debug output without ```sudo```**

```console
 ~/Dropbox/Application Support/dotfiles/Update  brew cask install music-manager --debug
==> Creating directories
==> Loading Cask definitions
==> Translating 'music-manager' into a valid Cask source
==> Testing source class Hbc::Source::URI
==> Testing source class Hbc::Source::PathSlashRequired
==> Testing source class Hbc::Source::TappedQualified
==> Testing source class Hbc::Source::UntappedQualified
==> Testing source class Hbc::Source::Tapped
==> Success! Using source class Hbc::Source::Tapped
==> Resolved Cask URI or file source to '/usr/local/Library/Taps/caskroom/homebrew-cask/Casks/music-manager.rb'
==> Cask instance dumps in YAML:
==> Cask instance toplevel:
--- !ruby/object:Hbc::Cask
token: music-manager
sourcefile_path: !ruby/object:Pathname
  path: "/usr/local/Library/Taps/caskroom/homebrew-cask/Casks/music-manager.rb"
dsl: !ruby/object:Hbc::DSL
  token: music-manager
  version: !ruby/string:Hbc::DSL::Version
    str: 1.0.243.1116
    raw_version: 1.0.243.1116
  sha256: e425d1724d092ae5ddfb28ad4304439754394b91ab3ff3063b66196d2ffe4bee
  url: !ruby/object:Hbc::URL
    uri: !ruby/object:URI::HTTPS
      scheme: https
      user: 
      password: 
      host: dl.google.com
      port: 443
      path: "/dl/androidjumper/mac/2431116/musicmanager.dmg"
      query: 
      opaque: 
      registry: 
      fragment: 
      parser: 
    user_agent: 
    cookies: 
    referer: 
    using: 
    revision: 
    trust_cert: 
    data: 
  name:
  - Google Play Music Manager
  homepage: https://play.google.com/music/
  license: !ruby/object:Hbc::DSL::License
    value: :unknown
  artifacts:
    :app: !ruby/object:Set
      hash:
        ? - MusicManager.app
          - :target: Music Manager.app
        : true
    :prefpane: !ruby/object:Set
      hash:
        ? - MusicManager.app/Contents/Helpers/MusicManager.prefPane
        : true
==> Cask instance method 'name':
---
- Google Play Music Manager
==> Cask instance method 'homepage':
--- https://play.google.com/music/
...
==> Cask instance method 'url':
--- !ruby/object:Hbc::URL
uri: !ruby/object:URI::HTTPS
  scheme: https
  user: 
  password: 
  host: dl.google.com
  port: 443
  path: "/dl/androidjumper/mac/2431116/musicmanager.dmg"
  query: 
  opaque: 
  registry: 
  fragment: 
  parser: 
user_agent: 
cookies: 
referer: 
using: 
revision: 
trust_cert: 
data: 
==> Cask instance method 'appcast':
--- 
...
==> Cask instance method 'version':
--- !ruby/string:Hbc::DSL::Version
str: 1.0.243.1116
raw_version: 1.0.243.1116
==> Cask instance method 'license':
--- !ruby/object:Hbc::DSL::License
value: :unknown
==> Cask instance method 'sha256':
--- e425d1724d092ae5ddfb28ad4304439754394b91ab3ff3063b66196d2ffe4bee
...
==> Cask instance method 'artifacts':
---
:app: !ruby/object:Set
  hash:
    ? - MusicManager.app
      - :target: Music Manager.app
    : true
:prefpane: !ruby/object:Set
  hash:
    ? - MusicManager.app/Contents/Helpers/MusicManager.prefPane
    : true
==> Cask instance method 'caveats':
--- []
==> Cask instance method 'depends_on':
--- 
...
==> Cask instance method 'conflicts_with':
--- 
...
==> Cask instance method 'container':
--- 
...
==> Cask instance method 'gpg':
--- 
...
==> Cask instance method 'accessibility_access':
--- 
...
==> Cask instance method 'auto_updates':
--- 
...
==> Hbc::Installer.install
==> Printing caveats
==> Downloading
==> Downloading https://dl.google.com/dl/androidjumper/mac/2431116/musicmanager.dmg
Already downloaded: /Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg
==> Downloaded to -> /Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg
==> Verifying download
==> Determining which verifications to run for Cask music-manager
==> Checking for verification class Hbc::Verify::Checksum
==> 1 verifications defined
Hbc::Verify::Checksum
==> Running verification of class Hbc::Verify::Checksum
==> Verifying checksum for Cask music-manager
==> SHA256 checksums match
==> Extracting primary container
==> Determining which containers to use based on filetype
==> Checking container class Hbc::Container::Pkg
==> Checking container class Hbc::Container::Ttf
==> Checking container class Hbc::Container::Otf
==> Checking container class Hbc::Container::Air
==> Checking container class Hbc::Container::Cab
==> Executing: ["/usr/bin/file", "-Izb", "--", "/Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg"]
==> Checking container class Hbc::Container::Dmg
==> Executing: ["/usr/bin/hdiutil", "imageinfo", "/Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg"]
==> Using container class Hbc::Container::Dmg for /Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg
==> Executing: ["/usr/bin/hdiutil", "mount", "-plist", "-nobrowse", "-readonly", "-noidme", "-mountrandom", "/tmp", "/Users/winkelsdorf/Library/Caches/Homebrew/music-manager-1.0.243.1116.dmg"]
==> Executing: ["/usr/bin/ditto", "--", "/private/tmp/dmg.bsA4El", "/opt/homebrew-cask/Caskroom/music-manager/1.0.243.1116"]
==> Executing: ["/usr/sbin/diskutil", "eject", "/private/tmp/dmg.bsA4El"]
==> Installing artifacts
==> Determining which artifacts are present in Cask music-manager
==> Checking for artifact class Hbc::Artifact::PreflightBlock
==> Checking for artifact class Hbc::Artifact::NestedContainer
==> Checking for artifact class Hbc::Artifact::Installer
==> Checking for artifact class Hbc::Artifact::App
==> Checking for artifact class Hbc::Artifact::Suite
==> Checking for artifact class Hbc::Artifact::Artifact
==> Checking for artifact class Hbc::Artifact::Colorpicker
==> Checking for artifact class Hbc::Artifact::Pkg
==> Checking for artifact class Hbc::Artifact::Prefpane
==> Checking for artifact class Hbc::Artifact::Qlplugin
==> Checking for artifact class Hbc::Artifact::Font
==> Checking for artifact class Hbc::Artifact::Service
==> Checking for artifact class Hbc::Artifact::StageOnly
==> Checking for artifact class Hbc::Artifact::Binary
==> Checking for artifact class Hbc::Artifact::InputMethod
==> Checking for artifact class Hbc::Artifact::InternetPlugin
==> Checking for artifact class Hbc::Artifact::AudioUnitPlugin
==> Checking for artifact class Hbc::Artifact::VstPlugin
==> Checking for artifact class Hbc::Artifact::Vst3Plugin
==> Checking for artifact class Hbc::Artifact::ScreenSaver
==> Checking for artifact class Hbc::Artifact::Uninstall
==> Checking for artifact class Hbc::Artifact::PostflightBlock
==> Checking for artifact class Hbc::Artifact::Zap
==> 2 artifact/s defined
Hbc::Artifact::App
Hbc::Artifact::Prefpane
==> Installing artifact of class Hbc::Artifact::App
==> Moving App 'MusicManager.app' to '/Applications/Music Manager.app'
==> Installing artifact of class Hbc::Artifact::Prefpane
==> Purging files for version 1.0.243.1116 of Cask music-manager
Error: It seems the Preference Pane source is not there: '/opt/homebrew-cask/Caskroom/music-manager/1.0.243.1116/MusicManager.app/Contents/Helpers/MusicManager.prefPane'/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:77:in `preflight_checks'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:24:in `block in install_phase'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:43:in `block in each_artifact'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:42:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:42:in `each_artifact'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/artifact/moved.rb:22:in `install_phase'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:129:in `block in install_artifacts'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:126:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:126:in `install_artifacts'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/installer.rb:75:in `install'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/install.rb:20:in `block in install_casks'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/install.rb:17:in `each'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/install.rb:17:in `install_casks'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli/install.rb:6:in `run'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:83:in `run_command'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:121:in `process'
/usr/local/Library/Taps/caskroom/homebrew-cask/cmd/brew-cask.rb:26:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Library/brew.rb:22:in `require?'
/usr/local/Library/brew.rb:93:in `<main>'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:126:in `exit'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:126:in `rescue in process'
/usr/local/Library/Taps/caskroom/homebrew-cask/lib/hbc/cli.rb:117:in `process'
/usr/local/Library/Taps/caskroom/homebrew-cask/cmd/brew-cask.rb:26:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/usr/local/Library/brew.rb:22:in `require?'
/usr/local/Library/brew.rb:93:in `<main>'
```

**Cask doctor output (latest versions as of posting this)**

```console
 ✘  ~/Dropbox/Application Support/dotfiles/Update  brew cask doctor
==> OS X Release:
10.11
==> OS X Release with Patchlevel:
10.11.5
==> Hardware Architecture:
intel-64
==> Ruby Version:
2.0.0-p648
==> Ruby Path:
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
==> Homebrew Version:
Homebrew 0.9.9 (git revision 656c; last commit 2016-06-03)
Homebrew/homebrew-core (git revision 8611; last commit 2016-06-03)
==> Homebrew Executable Path:
/usr/local/bin/brew
==> Homebrew Cellar Path:
/usr/local/Cellar
==> Homebrew Repository Path:
/usr/local
==> Homebrew Origin:
https://github.com/Homebrew/brew
==> Homebrew-cask Version:
0.60.0 (git revision 9c6ad; last commit 21 minutes ago)
==> Homebrew-cask Install Location:
<NONE>
==> Homebrew-cask Staging Location:
/opt/homebrew-cask/Caskroom
==> Homebrew-cask Cached Downloads:
/Users/winkelsdorf/Library/Caches/Homebrew
/Users/winkelsdorf/Library/Caches/Homebrew/Casks
0 files, 0B
==> Homebrew-cask Default Tap Path:
/usr/local/Library/Taps/caskroom/homebrew-cask
==> Homebrew-cask Alternate Cask Taps:
/usr/local/Library/Taps/alehouse/homebrew-unofficial
/usr/local/Library/Taps/caskroom/homebrew-fonts
/usr/local/Library/Taps/caskroom/homebrew-versions
==> Homebrew-cask Default Tap Cask Count:
3194
==> Contents of $LOAD_PATH:
/usr/local/Library/Taps/caskroom/homebrew-cask/lib
/usr/local/Library/Homebrew
/Library/Ruby/Gems/2.0.0/gems/psych-2.0.17/lib
/Library/Ruby/Site/2.0.0
/Library/Ruby/Site/2.0.0/x86_64-darwin15
/Library/Ruby/Site/2.0.0/universal-darwin15
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/x86_64-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/universal-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/x86_64-darwin15
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/universal-darwin15
==> Contents of $RUBYLIB Environment Variable:
<NONE>
==> Contents of $RUBYOPT Environment Variable:
<NONE>
==> Contents of $RUBYPATH Environment Variable:
<NONE>
==> Contents of $RBENV_VERSION Environment Variable:
<NONE>
==> Contents of $CHRUBY_VERSION Environment Variable:
<NONE>
==> Contents of $GEM_HOME Environment Variable:
<NONE>
==> Contents of $GEM_PATH Environment Variable:
<NONE>
==> Contents of $BUNDLE_PATH Environment Variable:
<NONE>
==> Contents of $PATH Environment Variable:
PATH="/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/winkelsdorf/.composer/vendor/bin:/usr/local/Library/Taps/caskroom/homebrew-cask/cmd:/usr/local/Library/ENV/scm"
==> Contents of $SHELL Environment Variable:
SHELL="/usr/local/bin/zsh"
==> Contents of Locale Environment Variables:
LANG="de_DE.UTF-8"
LC_CTYPE="de_DE.UTF-8"
==> Running As Privileged User:
No
```

**Comment by vitorgalvao** [comment]
Two small notes:

1. `parallels-desktop` already had issues before (the linked issue is older than the recent changes), but that could very well be related to the cask itself (which had some problems that were already fixed).
2. The problem with this cask is a fun one. See how it says `Error: It seems the Preference Pane source is not there: '/opt/homebrew-cask/Caskroom/music-manager/1.0.243.1116/MusicManager.app/Contents/Helpers/MusicManager.prefPane'`? Well, it depends in what order we’re doing things (will have to check), If we moved the target from `app` before trying to get the target from `prefpane`, then it’s natural it can’t find it, because it would have already been moved. To fix that, the line would need to be `prefpane "#{appdir}/MusicManager.app/Contents/Helpers/MusicManager.prefPane"`. That said, I don’t see a reason why we should be taking out the prefpane ourselves. Presumably the app will do it, so I’ll remove that line.
